Switch to 213-EAP platform. Avoid obsolete VMOptions methods usage.

IKrukov-HORIS opened the pull request.

SerVB commented:
AFAIR we've decided that we will proceed when 213 is released. Please request my review before merging. For now I'm going out of this PR to clean the list of pending PRs a bit :)

IKrukov-HORIS commented:
> AFAIR we've decided that we will proceed when 213 is released. Please request my review before merging. For now I'm going out of this PR to clean the list of pending PRs a bit :)

Hi! 213 is released some time ago ). I've rebased the branch to the latest master and  added some code cleanups. Please review.

SerVB commented on projector-plugin/src/main/kotlin/org/jetbrains/projector/plugin/Utils.kt:
```diff
@@ -87,9 +88,27 @@ private fun parseVersions(content: String): Map<String, String> {
+}
+
+fun addRequiredOptions() {
+  try { // // for 2021.3 and later
```
```suggestion
  try { // for 2021.3 and later
```
